plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/util/PluginListParser.java

```diff
@@ -69,37 +69,32 @@
     public List<Plugin> parsePluginYamlFile(File pluginYamlFile) {
         List<Plugin> pluginsFromYaml = new ArrayList<>();
         if (fileExists(pluginYamlFile)) {
-            Yaml yaml = new Yaml();
+            Representer representer = new Representer();
+            representer.getPropertyUtils().setSkipMissingProperties(true);
```

Comment:
Test will fail without ignoring missing properties 👍 Especially the bad format tests